pyiron_atomistics/vasp/base.py

```diff
@@ -2026,6 +2026,7 @@ def __init__(self):
         self.outcar = Outcar()
         self.oszicar = Oszicar()
         self.generic_output = GenericOutput()
+        self.dft_output = DFTOutput()
```

Comment:
I feel this is not included in the `to_dict()` and `from_dict()` as the VASP base implementation stores the corresponding entries in `Output.generic_output.dft_log_dict` while the VASP interactive implementation uses `Output.dft_output.log_dict`.

Reply:
I'm not sure how it is handled, but the `vasp.interactive.Output` was literally not used as far as I could tell.  When I create a test vasp job in a notebook (`pr.create.job.Vasp`) its `job._output_parser` object is the `vasp.base.Output`, only when `VaspInteractive.interactive_close` is called this was overwritten.  I can test again with a "proper" interactive vasp job, though, in case there are any weird edge cases.